app/views/transport/equipment/RemoveItemView.scala.html

```diff
@@ -0,0 +1,53 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.transport.equipment.routes
+@import viewModels.components.InputYesNoViewModel._
+@import views.html.components.{Button, ErrorSummary, InputYesNo, InsetText}
+@import models.reference.Item
+
+@this(
+mainTemplate: MainTemplate,
+formHelper: FormWithCSRF,
+inputYesNo: InputYesNo,
+errorSummary: ErrorSummary,
+        inset: InsetText,
+button: Button
+)
+
+@(form: Form[Boolean], departureId: String, mode: Mode, equipmentIndex: Index, itemIndex: Index, item: Item)(implicit request: Request[_], messages: Messages)
+
+@mainTemplate(
+    title = messages("transport.equipment.removeItem.title", equipmentIndex.display),
+    showBackLink = true
+) {
+
+    @formHelper(action = routes.RemoveItemController.onSubmit(departureId, mode, equipmentIndex, itemIndex), Symbol("autoComplete") -> "off") {
+
+        @errorSummary(form.errors)
+
+        @inputYesNo(
+            field = form("value"),
+            yesNoType = YesNoWithAdditionalHtml(
+                heading = messages("transport.equipment.removeItem.heading", equipmentIndex.display),
+                caption = Some(messages("transport.caption")),
+                additionalHtml = inset(messages("transport.equipment.removeItem.inset", itemIndex.display, item.description))
```

Comment:
Instead of the two params, you'll probably want call .toString on the item. I was speaking to Gavin about this when looking at the add another page. It looks like the structure is s"$goodsItemNumber - $description" rather than using the item index